darwin/importer/formats/coco.py

```diff
@@ -141,11 +142,29 @@ def parse_annotation(annotation: Dict[str, Any], category_lookup_table: Dict[str
                 path.append({"x": x, "y": y})
             except StopIteration:
                 break
-        return dt.make_polygon(category["name"], path)
+        return dt.make_polygon(category["name"], path, subs=subs)
     else:
         return None
 
 
+def _build_sub_annotations(annotation: dict):
+    """Builds the SubAnnotations from the annotation attributes.
+
+    Args:
+        annotation (Dict): The annotation dictionary.
+
+    Returns:
+        List[SubAnnotation]: SubAnnotation list or None if no extra info is
+        available.
+    """
+    extra_info = annotation.get("extra")
+    attributes = extra_info.get("attributes")
+    sub_ann = None
+    if extra_info and attributes:
+        sub_ann = [dt.make_attributes(attributes)]
+    return sub_ann
```

Comment:
I have seen coco files missing the `extra` field, so we need check that it exists before looking for attributes. 

```suggestion
    extra_info = annotation.get("extra")
    if not extra_info:
        return None
    attributes = extra_info.get("attributes")
    if not attributes:
        return None
    return [dt.make_attributes(attributes)]
```